### ISSUE TITLE
Can't delete selected value (stats)

### ISSUE THREAD

**Issue by LouisCuvelier** [nystudio107/craft-retour#85]
Hi,

I'm having this error when I try to delete stats values.

`Return value of nystudio107\retour\controllers\StatisticsController::actionDeleteStatistics() must be an instance of yii\web\Response, null returned`

**Comment by khalwat** [comment]
@LouisCuvelier check your `web.log` for errors?

**Comment by LouisCuvelier** [comment]
Does it help ?

```log
2019-11-19 00:00:39 [-][1][28a450asaelrusorrif5eof3nb][error][TypeError] TypeError: Return value of nystudio107\retour\controllers\StatisticsController::actionDeleteStatistics() must be an instance of yii\web\Response, null returned in /home/coursierjob/webapps/coursierjob/vendor/nystudio107/craft-retour/src/controllers/StatisticsController.php:167
Stack trace:
#0 [internal function]: nystudio107\retour\controllers\StatisticsController->actionDeleteStatistics()
#1 /home/coursierjob/webapps/coursierjob/vendor/yiisoft/yii2/base/InlineAction.php(57): call_user_func_array(Array, Array)
#2 /home/coursierjob/webapps/coursierjob/vendor/yiisoft/yii2/base/Controller.php(157): yii\base\InlineAction->runWithParams(Array)
#3 /home/coursierjob/webapps/coursierjob/vendor/craftcms/cms/src/web/Controller.php(187): yii\base\Controller->runAction('delete-statisti...', Array)
#4 /home/coursierjob/webapps/coursierjob/vendor/yiisoft/yii2/base/Module.php(528): craft\web\Controller->runAction('delete-statisti...', Array)
#5 /home/coursierjob/webapps/coursierjob/vendor/craftcms/cms/src/web/Application.php(299): yii\base\Module->runAction('retour/statisti...', Array)
#6 /home/coursierjob/webapps/coursierjob/vendor/craftcms/cms/src/web/Application.php(565): craft\web\Application->runAction('retour/statisti...', Array)
#7 /home/coursierjob/webapps/coursierjob/vendor/craftcms/cms/src/web/Application.php(278): craft\web\Application->_processActionRequest(Object(craft\web\Request))
#8 /home/coursierjob/webapps/coursierjob/vendor/yiisoft/yii2/base/Application.php(386): craft\web\Application->handleRequest(Object(craft\web\Request))
#9 /home/coursierjob/webapps/coursierjob/web/index.php(21): yii\base\Application->run()
#10 {main}
```

**Comment by khalwat** [comment]
Fixed in https://github.com/nystudio107/craft-retour/commit/b0650c9d0b464bf22e4e4b5efc4de320a1c62095

**Comment by LouisCuvelier** [comment]
Wow that's was fast ! Thank you !

**Comment by khalwat** [comment]
I happened to be in there already. Released as 3.1.29 -> https://github.com/nystudio107/craft-retour/releases/tag/3.1.29

lmk how you go

**Comment by LouisCuvelier** [comment]
Perfect !
